Mlem/App/Views/Root/Tabs/Feeds/FeedsView.swift

```diff
@@ -116,28 +116,13 @@ struct MinimalPostFeedView: View {
                 ScrollToView(appeared: $scrollToTopAppeared)
                     .id(scrollToTop)
                 
+                Divider()
                 ForEach(postTracker.items, id: \.uid) { post in
                     NavigationLink(value: NavigationPage.expandedPost(post)) {
-                        HStack {
-                            actionButton(post.upvoteAction)
-                            actionButton(post.downvoteAction)
-                            actionButton(post.saveAction)
-                            
-                            Text(post.title)
-                                .frame(maxWidth: .infinity, alignment: .leading)
-                                .padding(.horizontal)
-                                .foregroundStyle(post.read ? .secondary : .primary)
-                        }
-                        .padding(10)
-                        .background(palette.background)
-                        .contentShape(.rect)
-                        .contextMenu {
-                            ForEach(post.menuActions.children, id: \.id) { action in
-                                MenuButton(action: action)
-                            }
-                        }
+                        FeedPostView(post: .init(post: post))
+                            .contentShape(.rect)
                     }
-                    .buttonStyle(.plain)
+                    .buttonStyle(EmptyButtonStyle())
                     Divider()
```

Comment:
Kinda unrelated, but wrapping the `NavigationLink` and `Divider` in a `VStack(spacing: 0)` results in a significant performance boost